File: 1074/1074.c
#include<stdio.h>

int main(){

    int i, numeros , valor; 

    scanf("%d", &numeros);
    int vetor[numeros];

    for(i = 0 ; i<=numeros-1 ; i++){

       scanf("%d", &vetor[i]);
    }// for

    for(i = 0; i< numeros ; i++){
        
       valor = vetor[i];

       if(valor != 0){
           
           if(valor % 2 == 0){

               if(valor >= 1){
                   printf("EVEN POSITIVE\n");
               }
               else if(valor <= -1){
                   printf("EVEN NEGATIVE\n");
               }
           }//2º if
           else{
               if(valor >= 1){
                   printf("ODD POSITIVE\n");
               }
               else if(valor <= -1){
                   printf("ODD NEGATIVE\n");
               }
           }
       }
       else{
           printf("NULL\n");
       }

    }//forzão

    return 0;
}

File: 1074/1074.py
cont = int(input())
lista = []
for _ in range(0,cont):
    inp = int(input())
    lista.append(inp)

for percorre in lista:

    if(percorre != 0):
           
        if(percorre % 2 == 0):

            if(percorre >= 1):
                print("EVEN POSITIVE\n")
                    
            elif(percorre <= -1):
                print("EVEN NEGATIVE\n") 
                #2º if
        else:
            if(percorre >= 1):
                print("ODD POSITIVE\n")
                    
            elif(percorre <= -1):
                print("ODD NEGATIVE\n")
                    
    else:
        print("NULL\n")
       